### PR TITLE
Plot with Scanpy: Changes way in which coloring defaults are set (smallfix)

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This is a small patch to https://github.com/galaxyproject/tools-iuc/pull/3039

### CODE REVIEW

File: tools/scanpy/plot.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
     </stdio>
     <expand macro="version_command"/>
     <command detect_errors="exit_code"><![CDATA[
+#set plot = True
```
Comment (pcm32) [inline tools/scanpy/plot.xml:12]: Cool, I wasn't aware of this mechanism to couple errors to actions in other parts!

**Comment by bgruening** [comment]
```
 FAIL: ( scanpy_filter ) > Test-1
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/tmp9qdu23fq/galaxy-dev/test/functional/test_toolbox.py", line 99, in test_tool
    self.do_it(tool_version=tool_version, test_index=test_index)
  File "/tmp/tmp9qdu23fq/galaxy-dev/test/functional/test_toolbox.py", line 36, in do_it
    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)
  File "/tmp/tmp9qdu23fq/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py", line 789, in verify_tool
    raise e
  File "/tmp/tmp9qdu23fq/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py", line 785, in verify_tool
    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)
  File "/tmp/tmp9qdu23fq/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py", line 1006, in _verify_outputs
    raise JobOutputsError(found_exceptions, job_stdio)
galaxy.tool_util.verify.interactor.JobOutputsError: Standard output of the job different than expected
No text matching expression '336 × 11' was found in output file.
```

**Comment by pcm32** [comment]
I don't understand, does the plot wrapper uses scanpy_filter for testing it? How can I help with that?

**Comment by mtekman** [comment]
it's likely that '$plot' is not a very specific keyword and that it's being interpreted differently in filter

I am debugging both now

**Comment by mtekman** [comment]
I've removed the $plot statement and defaulted back to how it was before. Looking at the failing plot function it claims to be colorspace issue, which is strange since the test does not specify anything out of the ordinary.

It passes locally for me, but fails in docker. Shall we just skip this one test for expedience?

**Comment by mtekman** [comment]
oh I found the issue -- there's a default color map being specified

**Comment by bgruening** [comment]
You can use `planemo test --docker` to test inside of Docker. We enabled container testing to find those nasty font issues or any other stuff that might need xlibs etc ...

**Comment by mtekman** [comment]
Yep I use docker by default, but it's harder to debug

**Comment by bgruening** [comment]
Thanks @mtekman!

**Comment by bgruening** [comment]
Thanks @pcm32!

**Comment by pcm32** [comment]
Thanks to both of you!

**Comment by pcm32** [comment]
Do you know why deploy was skipped @bgruening ? I'm assuming that that is pushing it to the toolshed, correct? or is this done by some other CI process? Thanks!

**Comment by bgruening** [comment]
Its still running: https://github.com/galaxyproject/tools-iuc/actions/runs/110343342

**Comment by pcm32** [comment]
There might be some issues with the toolshed, the deploy part is taking more than 50 minutes now https://github.com/galaxyproject/tools-iuc/runs/692701703?check_suite_focus=true

**Pull request merged**